extra/utils/permissions.js

```diff
@@ -0,0 +1,29 @@
+let permissions={
+    'getUsers': {
+    all: ['head-trainer'],
+    read : ['trainee', 'trainer'],
+    write : ['trainer'],
+    delete: [],
+    }
+    }
+    function hasPermission(moduleName, role, permissionType) {
+        if (
+        permissions[moduleName]["all"].includes(role) ||
+        permissions[moduleName][permissionType].includes(role))
+        {
+        console.log("True");
+        return ;
+        } 
+        else 
```

Comment:
@rohit-ns There is a catch in your code, "return" in "else" block is of no use because you have written "return" in "if".
In short, no need to write "else" if you have "return" in "if" block.

Reply:
okk sir i have resolved this